notification/src/lambda.js

```diff
@@ -1,37 +1,57 @@
-const ctp = require('./utils/ctp')
 const handler = require('./handler/notification/notification.handler')
-const config = require('./config/config')()
+const config = require('./config/config')
 const logger = require('./utils/logger').getLogger()
-const { getNotificationForTracking } = require('./utils/commons')
 const setup = require('./config/init/ensure-interface-interaction-custom-type')
 
-const ctpClient = ctp.get(config)
+const { getNotificationForTracking } = require('./utils/commons')
 
 let initialised = false
 
 exports.handler = async function (event) {
   const { notificationItems } = event
-  try {
-    if (!initialised) {
-      await setup.ensureInterfaceInteractionCustomType(ctpClient)
-      initialised = true
-    }
-    if (!notificationItems) {
-      throw new Error('No notification received.')
-    }
-
-    await handler.processNotifications(notificationItems, ctpClient)
-    return {
-      notificationResponse: '[accepted]',
-    }
-  } catch (e) {
+  if (!notificationItems) {
+    const error = new Error('No notification received.')
     logger.error(
-      {
-        notification: getNotificationForTracking(notificationItems),
-        err: e,
-      },
-      'Unexpected error when processing event'
+        {
+          notification: getNotificationForTracking(notificationItems),
```

Comment:
As `notificationItems` is not defined here, you could just skip this line or put `undefined´

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/03573b810960539266e4e366286cca06cfa02cd8